tools/run_tests/artifacts/build_artifact_python.sh

```diff
@@ -81,6 +81,21 @@ then
   export GRPC_BUILD_OVERRIDE_BORING_SSL_ASM_PLATFORM="linux-arm"
 fi
 
+ancillary_package_dir=(
+  "src/python/grpcio_admin/"
```

Comment:
It would be great if we could have a single source of truth for the list here and the list of packages we're building lower in the file, but that would probably be more difficult than is reasonable for now.